types/node/stream.d.ts

```diff
@@ -1411,14 +1411,12 @@ declare module "stream" {
             writable?: boolean | undefined;
         }
         /**
-         * A readable and/or writable stream/webstream.
-         *
          * A function to get notified when a stream is no longer readable, writable
          * or has experienced an error or a premature close event.
          *
          * ```js
-         * import { finished } from 'node:stream';
-         * import fs from 'node:fs';
+         * const { finished } = require('node:stream');
+         * const fs = require('node:fs');
```

Comment:
Please do not make unrelated modifications.

Reply:
Done.